app/build.gradle

```diff
@@ -87,55 +66,118 @@ android {
+    }
+
+    testOptions {
+        unitTests {
+            includeAndroidResources = true
+        }
+    }
 }
 
 dependencies {
     implementation fileTree(include: ['*.jar'], dir: 'libs')
-    implementation project(':seekarc')
     implementation project(':exoplayer-flac')
+    implementation project(':seekarc')
 
-    testImplementation 'junit:junit:4.12'
-    testImplementation 'org.robolectric:robolectric:3.5.1'
-    testImplementation 'org.robolectric:shadows-support-v4:3.3.2'
-
+    // Support libraries
+    implementation "com.android.support:design:${rootProject.ext.supportLibVersion}"
+    implementation "com.android.support:support-v4:${rootProject.ext.supportLibVersion}"
+    implementation "com.android.support:appcompat-v7:${rootProject.ext.supportLibVersion}"
+    implementation "com.android.support:cardview-v7:${rootProject.ext.supportLibVersion}"
+    implementation "com.android.support:palette-v7:${rootProject.ext.supportLibVersion}"
+    implementation "com.android.support:recyclerview-v7:${rootProject.ext.supportLibVersion}"
+    implementation "com.android.support:preference-v14:${rootProject.ext.supportLibVersion}"
+    implementation 'com.android.support.constraint:constraint-layout:1.1.3'
+
+    implementation "android.arch.persistence.room:runtime:${rootProject.ext.roomVersion}"
+    annotationProcessor "android.arch.persistence.room:compiler:${rootProject.ext.roomVersion}"
+    testImplementation "android.arch.persistence.room:testing:${rootProject.ext.roomVersion}"
+
+    /** Dagger */
+    // TODO: Update to latest version (https://github.com/google/dagger/releases)
+    implementation 'com.google.dagger:dagger:2.6.1' //2.20
+    annotationProcessor 'com.google.dagger:dagger-compiler:2.6.1' //2.20
+
+    /** ReactiveX */
+    // TODO: Migrate to RxJava2 (https://github.com/ReactiveX/RxJava) (https://github.com/ReactiveX/RxAndroid)
+    implementation 'io.reactivex:rxjava:1.1.10' //1.3.8 //(https://github.com/ReactiveX/RxJava/blob/1.x/CHANGES.md)
+    //implementation 'io.reactivex.rxjava2:rxjava:2.2.4' //(https://github.com/ReactiveX/RxJava/blob/2.x/CHANGES.md)
+    implementation 'io.reactivex:rxandroid:1.2.1' //(https://github.com/ReactiveX/RxAndroid/blob/1.x/CHANGES.md)
+    //implementation 'io.reactivex.rxjava2:rxandroid:2.1.0' //(https://github.com/ReactiveX/RxAndroid/blob/2.x/CHANGES.md)
+
+    /** RxLifecycle */
+    /* TODO: Update RxLifecycle to latest version, migrate to RxLifecycle2, then migrate to RxLifecycle3
+     TODO: (https://github.com/trello/RxLifecycle/blob/master/CHANGELOG.md) */
+    implementation 'com.trello:rxlifecycle:0.6.1' //1.0
+    //implementation 'com.trello.rxlifecycle2:rxlifecycle:2.2.2'
+    //implementation 'com.trello.rxlifecycle3:rxlifecycle:3.0.0'
+    implementation 'com.trello:rxlifecycle-components:0.6.1' //1.0
+    //implementation 'com.trello.rxlifecycle2:rxlifecycle-components:2.2.2'
+    //implementation 'com.trello.rxlifecycle3:rxlifecycle-components:3.0.0'
```

Comment:
Let's drop all of the commented-out dependencies like this. Right now I'm not leaning towards using RxLifecycle after the RxJava2 migration, so this may not end up being used.